libs/api/metadata-converter/src/lib/gn4/gn4.field.mapper.ts

```diff
@@ -451,6 +451,34 @@ export class Gn4FieldMapper {
         }),
       }
     },
+    resourceIdentifier: (output, source) => {
+      const identifiers = getAsArray(selectField(source, 'resourceIdentifier'))
+
+      if (!identifiers.length) return output
+
+      const mappedIdentifiers = identifiers
+        .map((id) => {
+          const code = selectField<string>(id, 'code')
+          if (!code) return null
```

Comment:
does that really happen? I'm pretty sure we can assume that there will always be a code